WooCommerce/src/main/kotlin/com/woocommerce/android/ui/bookings/BookingMapper.kt

```diff
@@ -0,0 +1,69 @@
+package com.woocommerce.android.ui.bookings
+
+import com.woocommerce.android.ui.bookings.compose.BookingAppointmentDetailsModel
+import com.woocommerce.android.ui.bookings.compose.BookingAttendanceStatus
+import com.woocommerce.android.ui.bookings.compose.BookingStatus
+import com.woocommerce.android.ui.bookings.compose.BookingSummaryModel
+import com.woocommerce.android.ui.bookings.list.BookingListItem
+import com.woocommerce.android.util.CurrencyFormatter
+import org.wordpress.android.fluxc.persistence.entity.BookingEntity
+import java.time.Duration
+import java.time.ZoneOffset
+import java.time.format.DateTimeFormatter
+import java.time.format.FormatStyle
+import javax.inject.Inject
+
+class BookingMapper @Inject constructor(
+    private val currencyFormatter: CurrencyFormatter,
+) {
+    private val summaryDateFormatter: DateTimeFormatter = DateTimeFormatter.ofLocalizedDateTime(
+        FormatStyle.MEDIUM,
+        FormatStyle.SHORT
+    ).withZone(ZoneOffset.UTC)
+
+    private val detailsDateFormatter: DateTimeFormatter = DateTimeFormatter.ofPattern("EEEE, dd MMM yyyy")
```

Comment:
IMO we should avoid hardcording formatting pattens, I'd use a localized format instead. I know it's used in the app is some spots, but IMO it's not something we should follow.


```suggestion
    private val detailsDateFormatter: DateTimeFormatter = DateTimeFormatter.ofLocalizedDate(FormatStyle.FULL)
```

The above will update the text from: `Monday, 06 Oct 2025` to `Monday, 6 October 2025` for UK english set as the locale.

Another approach if we are sure we want to keep the shorter month name is to use something like:

```kotlin
    private val detailsDateFormatter: DateTimeFormatter = DateTimeFormatterBuilder()
        .appendPattern("EEEE, ")
        .append(DateTimeFormatter.ofLocalizedDate(FormatStyle.MEDIUM))
        .toFormatter()
        .withZone(ZoneOffset.UTC)
```

But personally I think using `FormatStyle.FULL` is better in this context, WDYT?

Reply:
Makes sense, I did focus on the format I saw in the Figma, but this will work better across different locales. Thanks!